exporter/elasticsearchexporter/internal/exphistogram/exphistogram.go

```diff
@@ -41,12 +41,12 @@ func ToTDigest(dp pmetric.ExponentialHistogramDataPoint) (counts []int64, values
 		}
 		lb := -LowerBoundary(offset+i+1, scale)
 		ub := -LowerBoundary(offset+i, scale)
-		counts = append(counts, int64(count))
+		counts = append(counts, safeUint64ToInt64(count))
```

Comment:
In this particular case, I don't mind changing `counts` to []uint64 but up the call chain you'll find OTel doesn't support Uint64 slice inside a map IIUC.

Reply:
Yeah, and we can't change pdata anyway, as that would be a breaking change over all of the collector.